src/background.js

```diff
@@ -523,6 +523,12 @@ async function checkNormandyAddonStudy() {
 
   const branch = study.branch;
   switch (branch) {
+  case "helper":
+    // This case is not return true/false as its user branch contains users from
+    // both the experiement and the control branches. However, this function
+    // in conjunction with migrateLocalStoragePrefs() will turn on for the users
+    // who should, and not for those who shouldn't.
+    return null;
```

Comment:
Returning `null` here is technically ok since `null` and `false` are both equivalent to boolean `false` in JavaScript.

However, semantically this function is returning whether or not the Normandy study state is mandating us to turn this on even if we don't have any other reason to. In that sense, "false" seems fine to me. Perhaps the best thing to do is to write a comment to document the function above its definition that describes in what circumstances it returns true.

I'd prefer returning false over returning null, but like I said they are both semantically identical in this code.

Reply:
BTW, I like that this is throwing an error if we are in a study but we haven't explicitly detected it in the switch block.